app/templates/interstitial.html

```diff
@@ -1,2 +1,62 @@
-{% extends 'questionnaire.html' %}
-{% set submit_btn_text = "Continue" %}
+{% extends theme('layouts/_twocol.html') %}
```

Comment:
@ajmaddaford and I tried doing it this way and using a block, similar to the submit button block, but for some reason it wouldn't work?